cinema/tests/test_movie_api.py

```diff
@@ -54,9 +54,7 @@ def test_get_movies(self):
     def test_retrieve_movie(self):
```

Comment:
add typehint

Reply:
Do we have to add type hints to the tests which wasn't written by us?

Reply:
yeah, just do it